Raven.Database/Raft/ClusterKeeperTask.cs

```diff
@@ -145,7 +145,7 @@ private void HandleClusterReplicationChanges(List<string> removedNodes, bool ena
 
                 if (destination != null && node == null)
                 {
-                    if (removedNodes.Contains(url, StringComparer.OrdinalIgnoreCase) == false)
+                    if (removedNodes != null && removedNodes.Contains(url, StringComparer.OrdinalIgnoreCase) == false)
```

Comment:
I think that you're right, I changed this.
This code is so confusing and I'm sure what is the case of node == null. Maybe we want to remove it from destination in any case once it is null.